docs/modules/ROOT/pages/how-tos/exoscale/add_storage_node.adoc

```diff
@@ -0,0 +1,172 @@
+= Add a storage node
+
+:kubectl_extra_args: --as=cluster-admin
+:delabel_app_nodes: yes
+:argo_app: rook-ceph
+
+[abstract]
+--
+Steps to add a storage node to an OpenShift 4 cluster on https://www.exoscale.com[Exoscale].
+--
+
+== Starting situation
+
+* You already have a OpenShift 4 cluster on Exoscale
+* You have admin-level access to the cluster
+* You want to add a new storage node to the cluster
+
+== Prerequisites
+
+The following CLI utilities need to be available locally:
+
+* `docker`
+* `curl`
+* `kubectl`
+* `oc`
+* `exo` >= v1.28.0 https://community.exoscale.com/documentation/tools/exoscale-command-line-interface[Exoscale CLI]
+* `vault` https://www.vaultproject.io/docs/commands[Vault CLI]
+* `commodore`, see https://syn.tools/commodore/running-commodore.html[Running Commodore]
+* `jq`
+* `yq` https://mikefarah.gitbook.io/yq[yq YAML processor] (version 4 or higher)
+
+== Prepare local environment
+
+. Create local directory to work in
++
+[TIP]
+====
+We strongly recommend creating an empty directory, unless you already have a work directory for the cluster you're about to work on.
+This guide will run Commodore in the directory created in this step.
+====
++
+[source,bash]
+----
+export WORK_DIR=/path/to/work/dir
+mkdir -p "${WORK_DIR}"
+pushd "${WORK_DIR}"
+----
+
+. Configure API access
++
+include::partial$exoscale/environment-vars.adoc[]
++
+include::partial$vshn-input.adoc[]
+
+. Get required tokens from Vault
++
+include::partial$connect-to-vault.adoc[]
++
+include::partial$get-hieradata-token-from-vault.adoc[]
++
+[source,bash]
+----
+export TF_VAR_lb_exoscale_api_user=$(vault kv get \
+  -format=json "clusters/kv/${TENANT_ID}/${CLUSTER_ID}/floaty" | jq '.data.data')
+export TF_VAR_lb_exoscale_api_key=$(echo "${TF_VAR_lb_exoscale_api_user}" | jq -r '.iam_key')
+export TF_VAR_lb_exoscale_api_secret=$(echo "${TF_VAR_lb_exoscale_api_user}" | jq -r '.iam_secret')
+----
```

Comment:
This is probably also required in `replace_storage_node.adoc`, not sure how I missed that.